packages/block-editor/src/components/block-mover/mover-description.js

```diff
@@ -49,47 +49,128 @@ export function getBlockMoverDescription( selectedCount, type, firstIndex, isFir
 
 	if ( dir > 0 && ! isLast ) {
 		// moving down
-		return sprintf(
-			// translators: 1: Type of block (i.e. Text, Image etc), 2: Position of selected block, 3: Direction of movement ( up, down, left, right ), 4: New position
-			__( 'Move %1$s block from position %2$d %3$s to position %4$d' ),
-			type,
-			position,
-			getMovementDirection( 'down' ),
-			( position + 1 ),
-		);
+		const movementDirection = getMovementDirection( 'down' );
+
+		if ( movementDirection === 'down' ) {
+			return sprintf(
+				// translators: 1: Type of block (i.e. Text, Image etc), 2: Position of selected block, 3: New position
+				__( 'Move %1$s block from position %2$d down to position %3$d' ),
+				type,
+				position,
+				( position + 1 ),
+			);
+		}
+
+		if ( movementDirection === 'left' ) {
+			return sprintf(
+				// translators: 1: Type of block (i.e. Text, Image etc), 2: Position of selected block, 3: New position
+				__( 'Move %1$s block from position %2$d left to position %3$d' ),
+				type,
+				position,
+				( position + 1 ),
+			);
+		}
+
+		if ( movementDirection === 'right' ) {
+			return sprintf(
+				// translators: 1: Type of block (i.e. Text, Image etc), 2: Position of selected block, 3: New position
+				__( 'Move %1$s block from position %2$d right to position %3$d' ),
+				type,
+				position,
+				( position + 1 ),
+			);
+		}
 	}
 
 	if ( dir > 0 && isLast ) {
 		// moving down, and is the last item
-		// translators: 1: Type of block (i.e. Text, Image etc), 2: Direction of movement ( up, down, left, right )
-		return sprintf(
-			__( 'Block %1$s is at the end of the content and can’t be moved %2$s' ),
-			type,
-			getMovementDirection( 'down' ),
+		// translators: 1: Type of block (i.e. Text, Image etc)
```

Comment:
This translators note should probably be in each of the `sprintf`s , just like it is in the other `if` blocks.

Reply:
Good catch!